### PR TITLE
League manager ar

### PR DESCRIPTION
**Pull request by garussell**
This PR added all but the last method for league manager.  It also added helper methods and added all new methods to stat tracker.

### CODE REVIEW

**Review by jpasko1112 — APPROVED** [review]
Really good work, thank you for you dedication and hard work.

File: lib/game_manager.rb [hunk 1]
```diff
@@ -125,4 +125,25 @@ def total_games_per_season(season)
   def average_scores_by_season(season)
     (total_goals_by_season(season).to_f / total_games_per_season(season)).round(2)
   end
+
+  # helper methods
```
Comment (danieljsauer) [inline lib/game_manager.rb:129]: Like this organization!

File: lib/stat_tracker.rb [hunk 2]
```diff
@@ -97,17 +97,43 @@ def average_scores_by_season
...
   def best_offense
     @league_manager.best_offense
   end
-
+  
   def worst_offense
     @league_manager.worst_offense
   end
+  
+  def lowest_scoring_visitor
```
Comment (danieljsauer) [inline lib/stat_tracker.rb:127]: These additions make sense

**Review by danieljsauer — APPROVED** [review]
Methods and organization all look good

**Pull request merged**